packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -310,6 +313,9 @@ - (void)updatePlayingState {
   }
   if (_isPlaying) {
     [_player play];
+
+    // fix to always set playback speed accurately
+    [self setPlaybackSpeed:_internalPlaybackSpeed];
```

Comment:
Is this always needed here? My impression from comments in the issue was that calling it from the completion handler of the seek method would likely be enough.

Reply:
@stuartmorgan which completion handler are you referring to? I only see a `seekTo` method in the file.

Reply:
See https://github.com/flutter/flutter/issues/71264#issuecomment-734656335

Reply:
@stuartmorgan I see. I tried moving the call inside the completionHandler block of the `seekTo` method as follows:

```
[_player seekToTime:CMTimeMake(location, 1000)
      toleranceBefore:kCMTimeZero
       toleranceAfter:kCMTimeZero
    completionHandler:^(BOOL finished) {
    [self setPlaybackSpeed:_internalPlaybackSpeed];
  }];
```

but the issue of the playback speed resetting to `1.0` came back. Therefore that is not equivalent.

Reply:
Interesting. So presumably then it's the `playbackLikelyToKeepUpContext` and/or `AVPlayerItemStatusReadyToPlay` codepath that's resolving this? In which case perhaps the underlying issue is that AVPlayer automatically resets the playback speed when the buffer is insufficient.